### PR TITLE
HomeWork Lists

### PR DESCRIPTION
**Pull request by qpuhuk**
HW done

### CODE REVIEW

File: src/main/java/AlexLink/HomeWork/HomeworkLists/Lists.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+public class Lists {
+    public static void main(String[] args) {
+
+        List<Integer> points = new ArrayList<>();
+        for (int i = 0; i < 10; i++) {
+            points.add(((int) (Math.random() * 11)));
+        }
+        System.out.println(points);
+
+                List<Integer> randomNumber = new ArrayList<>(points);
```
Comment (svetamolchanova) [inline src/main/java/AlexLink/HomeWork/HomeworkLists/Lists.java]: wrong tabulation and unnecessary space

File: src/main/java/AlexLink/HomeWork/HomeworkLists/Lists.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                randomNumber.remove(i - 2);
+                i--;
+            }
+        }
+        System.out.println(randomNumber);
+    }
+
+    @Override
+    public int hashCode() {
+        return super.hashCode();
```
Comment (svetamolchanova) [inline src/main/java/AlexLink/HomeWork/HomeworkLists/Lists.java]: why do you override methods hashcode and equals if use realization of class parent?

File: src/main/java/AlexLink/HomeWork/HomeworkLists/Lists.java (hunk 1 above)
Comment (qpuhuk) [reply src/main/java/AlexLink/HomeWork/HomeworkLists/Lists.java]: fixed it

File: src/main/java/AlexLink/HomeWork/HomeworkLists/Lists.java (hunk 2 above)
Comment (qpuhuk) [reply src/main/java/AlexLink/HomeWork/HomeworkLists/Lists.java]: fixed it

**Pull request merged**